src/main/java/com/northconcepts/datapipeline/examples/parquet/WriteAParquetFileUsingSchemaFromData.java

```diff
@@ -0,0 +1,50 @@
+package com.northconcepts.datapipeline.examples.parquet;
+
+import java.io.File;
+import java.sql.SQLException;
+
+import com.northconcepts.datapipeline.core.DataReader;
+import com.northconcepts.datapipeline.core.Record;
+import com.northconcepts.datapipeline.core.StreamWriter;
+import com.northconcepts.datapipeline.internal.jdbc.JdbcFacade;
+import com.northconcepts.datapipeline.jdbc.JdbcConnectionFactory;
+import com.northconcepts.datapipeline.jdbc.JdbcReader;
+import com.northconcepts.datapipeline.job.Job;
+import com.northconcepts.datapipeline.parquet.ParquetDataReader;
+import com.northconcepts.datapipeline.parquet.ParquetDataWriter;
+
+public class WriteAParquetFileUsingSchemaFromData {
+
+    public static void main(String[] args) throws SQLException {
+        Record record = new Record()
+                .setField("USER_ID", 11)
+                .setField("USER_NAME", "james_bond")
+                .setField("FIRST_NAME", "james")
+                .setField("LAST_NAME", "bond")
+                .setField("EMAIL", "james_bond@example.com")
+                .setField("PASSWORD", "*******************")
+                .setField("USER_ROLE_ID", 1)
+                ;
+
+        ParquetDataWriter writer = new ParquetDataWriter(new File("example/data/output/WriteAParquetFileUsingSchemaFromDatabase.parquet"));
+        // Set schema using a record
+        writer.setSchema(record);
```

Comment:
Let's rename this example to `WriteAParquetFileUsingSchemaFromRecord` and create a separate `WriteAParquetFileUsingSchemaFromData` that uses the entire dataset via `DynamicSchemaWriter` behind the scenes.